starters/README.md

```diff
@@ -78,7 +78,7 @@ npm install -g gatsby-cli
 gatsby new my-blazing-fast-site
 ```
 
-e.g. `gatsby new blog my-blazing-fast-site` or `gatsby new hello-world my-blazing-fast-site` to use a specific starter!
+e.g. `gatsby new my-blazing-fast-site blog` or `gatsby new my-blazing-fast-site hello-world` to use a specific starter!
```

Comment:
IIUC, the `starter` is still wrong, which should be like `<username>/<starter name>`.
```suggestion
e.g. `gatsby new my-blazing-fast-site gatsbyjs/gatsby-starter-blog` or `gatsby new my-blazing-fast-site gatsbyjs/gatsby-starter-hello-world` to use a specific starter!
```